oas_docs/bundle.serverless.json

```diff
@@ -46211,7 +46211,7 @@
         "tags": [
           "maintenance-window"
         ],
-        "x-state": "Generally available; added in 9.1.0"
+        "x-state": "Generally available"
```

Comment:
What caused all these diffs? Can we remove them from this PR?

Reply:
I have no idea what triggered them 😱  there's a convo on kibana-operations channel about it though.